lib/wor/requests/base.rb

```diff
@@ -85,35 +87,39 @@ def uri(path)
       end
 
       def formatted_base_url
+        raise NoMethodError, 'The method base_url is malformed' if base_url.nil?
```

Comment:
What do you think about to raise [a CustomError](https://github.com/Wolox/wor-requests/blob/master/lib/wor/requests/request_error.rb) here
```suggestion
        raise MalformedBaseUrl if base_url.nil?
```

```rb
  # wor-requests/lib/wor/requests/request_error.rb 

  class MalformedBaseUrl < StandardError
    def message
      'The method base_url is malformed'
    end
  end
```